.github/workflows/npm.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish npm package
```

Comment:
We could actually run npm publish via semantic release to follow suit with some other repos, but it will require some tweaks to the PR.
Specifically, you'll need to create a `.releaserc` file [like this one](https://github.com/hirosystems/chainhook/blob/develop/.releaserc), but add an entry for `@semantic-release/npm` before the `@semantic-release/github` entry.